Libs/MRML/DisplayableManager/vtkMRMLThreeDViewInteractorStyle.cxx

```diff
@@ -61,6 +64,29 @@ void vtkMRMLThreeDViewInteractorStyle::PrintSelf(ostream& os, vtkIndent indent)
   this->Superclass::PrintSelf(os,indent);
 }
 
+//----------------------------------------------------------------------------
+void vtkMRMLThreeDViewInteractorStyle::SetDisplayableManagers(vtkMRMLDisplayableManagerGroup* displayableManagers)
+{
+  if (displayableManagers == this->DisplayableManagers)
+    {
+    return;
+    }
+
+  this->Superclass::SetDisplayableManagers(displayableManagers);
+
+  // Observe ActiveCameraChangedEvent
+  vtkMRMLAbstractDisplayableManager* cameraDisplayableManager =
+      this->DisplayableManagers->GetDisplayableManagerByClassName("vtkMRMLCameraDisplayableManager");
+  if (cameraDisplayableManager)
+    {
+    // \tbd active camera should be set to view node instead and only observing
```

Comment:
`tbd` usually means "to be defined", do you mean `\todo`? (I could not find `\tbd` doxygen tag - https://www.doxygen.nl/manual/commands.html)

Reply:
I copied the comment from https://github.com/Slicer/Slicer/blob/9f95ec7deb1045167c59644a7634ecde5b60ac8a/Libs/MRML/DisplayableManager/vtkMRMLViewDisplayableManager.cxx#L652-L655

Using `\todo` should indeed better. I will update it at both location